pkg/controllers/machine_controller.go

```diff
@@ -173,6 +194,44 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 	return nil
 }
 
+func (r *MachineReconciler) processVolumeSizeChanges(parent context.Context, log logr.Logger) {
+	wait.UntilWithContext(parent, func(ctx context.Context) {
+		machines, err := r.machines.List(ctx)
+		if err != nil {
+			log.Error(err, "failed to list machines")
+			return
+		}
+
+		for _, machine := range machines {
+			var shouldEnqueue bool
+
+			for _, volume := range machine.Spec.Volumes {
+				plugin, err := r.volumePluginManager.FindPluginBySpec(volume)
+				if err != nil {
+					log.Error(err, "failed to get volume plugin", "machineID", machine, "volume", volume.Name)
```

Comment:
inconsistent key name volume (vs volumeName) with different part of code: https://github.com/ironcore-dev/libvirt-provider/pull/104/files#diff-7d2054388a9e1a4b7f89c284e697641903421c8fa3e63e30c15cacb9ab393979R157

Keeping of the same name can be useful for filtering of log records.